src/css/profile/mobile/common/progress.less

```diff
@@ -399,8 +417,8 @@ tau-progress {
 }
 
 .ui-listview .ui-li-has-progress {
-	height: 8 * @unit_base;
-	padding: (120 - 8) / 2 * @unit_base 0;
+	height: 8 * @px_base;
+	padding: (120 - 8) / 2 * @px_base 0;
```

Comment:
Isn't it good to use the calculated result right away?

Reply:
I think this is ok. Sometimes a developer want to describe the way to particular value instead of write "magic" value. This calculation will be done during compilation of less files. Alternatively, we could replace this place with a comment.

Reply:
I think that way is better when the code calculations are showed because developer can see why the result is like that. If we give the only number what is the solution, then it is difficult to edit when something changes.